android-pay/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
           package="com.stripe.wrap.pay"
           android:versionCode="1"
-          android:versionName="4.0.1">
```

Comment:
This should never have been such a large number -- i originally thought we'd have to keep them in sync.